README.md

```diff
@@ -64,6 +64,10 @@ Linux amd64 Tags
 | `4-powershell7-slim`                         |  Debian 10  |
 | `4-powershell7-appservice`                   |  Debian 10  |
 | `4-powershell7-core-tools`                   | Debian 10  |
+| `4-powershell7.2`                              |  Debian 10  |
+| `4-powershell7.2-slim`                         |  Debian 10  |
+| `4-powershell7.2-appservice`                   |  Debian 10  |
+| `4-powershell7.2-core-tools`                   | Debian 10  |
```

Comment:
I believe that Bullseye is the pseudonym for Debian 11 meaning that all of our Funcv4 apps are running on Debian 11 at this time

Reply:
fixed. Thanks!